applications/OptimizationApplication/python_scripts/utilities/opt_projection.py

```diff
@@ -1,3 +1,6 @@
+# Temporal fix until to avoid list[type] related errors until we move to 3.9 and above.
+from __future__ import annotations
```

Comment:
that is pretty nice to know

FYI @sunethwarna better use this and then e.g.  `list[sth]` instead of `List[sth]`, ` sth | None` instead of `Optional[std]` etc